src/concourse/pipelines/infrastructure/tika/packer_pulumi_pipeline.py

```diff
@@ -0,0 +1,68 @@
+from concourse.lib.constants import PULUMI_CODE_PATH
```

Comment:
Packer+Pulumi pipelines, however do lend themselves well to this framework and it was fairly easy to stamp out pipelines like this with minimal effort once I got going. 

I think we could go even further and condense many of these into one pipeline that is built based on a data structure that is something like this 

```
{ "tika" : { "project_source_path" : "applications/tika/",
           "watch_paths" : ["src/bilder/components/","src/bilder/images/tika/","src/bridge/lib/versions.py"],
           "stages": ["CI, "QA", "Production"],
           "other_piece_of_relevant_information": "relevant information"}
"vector_log_proxy": { ... } 
}
```